src/SimulationData.jl

```diff
@@ -4,7 +4,15 @@ using DataDeps
 using HDF5
 import Pkg
 
+export
+    read_estimate,
+    read_distribution,
+    read_array,
+    read_table
+
+
 include("units.jl")
+include("pyapi.jl")
```

Comment:
Hey, I think this file is missing from the PR?

Reply:
should be included now

Reply:
I don't get why it's not appearing in the files changed. It's in the file tree:

https://github.com/ScottishCovidResponse/SimulationData.jl/blob/ar/pycall/src/pyapi.jl

Reply:
Probably related to the outage today, maybe make a new PR and close this one?